doc/source/user_guide/label_guide.rst

```diff
@@ -4,42 +4,46 @@
 Label guide
 ===========
 
-Basic labeling
---------------
+Basic labelling
+---------------
 
-All ArviZ plotting functions and some stats functions take an optional ``labeller`` argument.
-By default, labels show the variable name and the coordinate value
-(for multidimensional variables only).
-The first example below uses this default labeling.
+All ArviZ plotting functions and some stats functions can take an optional ``labeller`` argument.
+By default, labels show the variable name.
+Multidimensional variables also show the coordinate value.
 
 .. ipython::
 
+Example: Default labelling
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
   In [1]: import arviz as az
      ...: schools = az.load_arviz_data("centered_eight")
      ...: az.summary(schools)
 
-Thanks to being powered by xarray, ArviZ supports label based indexing.
-We can therefore use the labels we have seen in the summary to plot only a subset of the variables,
-the one we are interested in.
-Provided we know that the coordinate values shown for theta correspond to the `school` dimension,
-we can plot only ``tau`` to better inspect it's 1.03 :func:`~arviz.rhat` and
-``theta`` for ``Choate`` and ``St. Paul's``, the ones with higher means:
+ArviZ supports label based indexing.
+Through label based indexing you can use labels to plot a subset of selected variables.
+
+Example: Label based indexing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For a case where the coordinate values shown for the theta variable correspond to the `school` dimension
+you can plot only the ``tau`` variable to better inspect its 1.03 :func:`~arviz.rhat` value,
+only the ``theta`` variable to the ``Choate`` and ``St. Paul's`` coordinates, which have higher means, you can ploy only the ``theta`` variable by running the following command:
```

Comment:
My original description wasn't very good, but it tried to say that we are plotting all the things that were described, we are not plotting only the `theta` variable. How about something like:

```
Given that summary by default shows the coordinate values and not their indices or some aliases, 
you can use what you see to subset the data. 
The summary shows that ``tau`` has 1.03 :func:`~arviz.rhat` value which deserves closer inspection. 
You can indicate ArviZ to plot ``tau`` by including it in the ``var_names`` argument. 
Moreover, you may also want to inspect closely the ``theta`` values for the ``Choate`` 
and ``St. Paul's`` coordinates because of their higher means. 
You can do so by including ``theta`` too in ``var_names`` and 
using the ``coords`` argument to select only these two coordinate values for the ``school`` dimension. 
You can generate this plot with the following command:
```